requirements.txt

```diff
@@ -37,6 +37,7 @@ drf-nested-routers==0.93.4
 drf-yasg==1.20.0
 environs==9.5.0
 gunicorn==20.1.0
+Markdown==3.4.1
```

Comment:
Can we make use of the `mistune` packages that we already have installed instead of adding another Markdown package?

I picked `mistune` for its performance and it seems to work well.

Reply:
Done in https://github.com/HyphaApp/hypha/pull/2991/commits/21c575af9a68dfddac8e9bdfed8edfb736fd1377